speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/MediatorTests.cpp

```diff
@@ -51,3 +51,44 @@ TEST_CASE(MediatorTests, InterventionTestNoSignals, InterventionTest, (INTERVENT
 TEST_CASE(MediatorTests, InterventionTestOnlySignals, InterventionTest, (INTERVENTION_TYPE_ONLY_SIGNALS))
 TEST_CASE(MediatorTests, InterventionTestSharedControl, InterventionTest, (INTERVENTION_TYPE_SHARED_CONTROL))
 TEST_CASE(MediatorTests, InterventionTestCompleteTakeover, InterventionTest, (INTERVENTION_TYPE_COMPLETE_TAKEOVER))
+
+/// @brief            Writes a given mediator pointer to the Singletons/Mediator file
+/// @param p_mediator The mediator pointer to write to the file
+bool WriteMediator(SMediator* p_mediator)
+{
+    EXPECT_TRUE(SetupSingletonsFolder());
+    struct stat info
+    {
+    };
```

Comment:
```suggestion
    struct stat info;
```

Reply:
This makes it so the struct isn't actually initialized with default values. clang-tidy will complain here.
Then clang-format turns it into three lines.

Reply:
This has been alternatively fixed: struct state info = {}; 
which is also compliant with clang-tidy, but does initialize the struct with default values.